### ISSUE TITLE
Impossible to reinitialize workspace

### ISSUE THREAD

**Issue by trivoallan** [nvie/gitflow#28]
Using Ubuntu 10.04 and code from develop branch (4de01f2)

```
$ git flow init -f
flags:WARN getopt: option invalide -- 'f'
 -- 'init'
flags:FATAL unable to parse provided options with getopt.
```

**Comment by ghost** [comment]
This is a known issue with shFlags on Ubuntu's getopt. Will have to see whether the shFlags project can pick this up, or whether we could provide a patch for the shFlags repo ourselves.

A workaround for now is to run the following commands:

```
git config --unset gitflow.branch.master
git config --unset gitflow.branch.develop
git config --unset gitflow.prefix.feature
git config --unset gitflow.prefix.release
git config --unset gitflow.prefix.hotfix
git config --unset gitflow.prefix.support
git config --unset gitflow.prefix.versiontag
```

Then call `git flow init` without the -f option.

**Comment by trivoallan** [comment]
Ok. Thanks for the  workaround. is there a place where i can vote for resolution of the upstream bug ?

**Comment by ghost** [comment]
I've only recently tested gitflow on Ubuntu, so it's fairly fresh. Haven't had the time to figure out how to translate this exactly to a bug report on shFlags. Help is appreciated with this one.

**Comment by nvie** [comment]
Decided not to invest in shFlags. Will do a complete reimplementation of gitflow in Python soon (http://github.com/nvie/gitflow/issues#issue/33), which won't suffer from the command line platform difficulties.

**Comment by nvie** [comment]
Fixed in e1ec57d48ac5c234262d84469af56b00b38929ab and confirmed on Ubuntu!

**Comment by vdelau** [comment]
Can confirm this fixed on Debian as well. (was broken in 0.4-dev)
